tools/release.sh

```diff
@@ -162,17 +162,17 @@ export GITHUB_USER="ankidroid"
 export GITHUB_REPO="Anki-Android"
 
 if [ "$PUBLIC" = "public" ]; then
-  PRE_RELEASE=""
+  RELEASE_TYPE="--latest"
 else
-  PRE_RELEASE="--pre-release"
+  RELEASE_TYPE="--pre-release"
```

Comment:
```suggestion
  RELEASE_TYPE="--prerelease"
```

From docs (thanks for linking those so I could easily open it to carefully verify everything...), it appears we were getting this wrong before, and it could use a correction as well:

https://cli.github.com/manual/gh_release_create

```
-p, --prerelease
Mark the release as a prerelease
```

That may even be the root cause of the malfunction, as the docs for `--latest` say:

> (default [automatic based on date and version])

...which means if we weren't marking pre-releases as such, it may not have been able to automatically figure out whether a release on an "older" version (e.g. 2.20.1 vs 2.21-alpha5) was latest or not. Just a guess on my part though, and with your change it will all be explicit not automatic / inferred, which is better

Reply:
Done!